lib/index.js

```diff
@@ -74,7 +74,13 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         if(err) {
           return next(err);
         }
-        res.status(201).location(baseUri + '/' + account.id).json(account);
+        // emit auto-login event for extensions to implement additional
+        // behavior after first account login
+        bedrock.events.emit('bedrock-account-http.autoLogin', {req, account})
```

Comment:
Add a FIXME to convert using async/await syntax in the future.

Reply:
Can't be converted because of passport API usage. I can leave a note about that though -- should passport ever fix their problem.